<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="b_answer">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>120</y>
      <width>80</width>
      <height>90</height>
     </rect>
    </property>
    <property name="text">
     <string>정 답</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_com3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>320</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>?</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_com4">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>323</y>
      <width>20</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>?</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_com2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>320</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>?</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_computerball">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>280</y>
      <width>80</width>
      <height>120</height>
     </rect>
    </property>
    <property name="text">
     <string>컴퓨터 
 BALL 
 공개</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="t_num4">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>120</y>
      <width>110</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="t_num2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>110</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="t_num1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>110</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="t_num3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>120</y>
      <width>110</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_reset">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>30</y>
      <width>90</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>처음으로</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>221</width>
      <height>91</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>190</red>
          <green>190</green>
          <blue>190</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <pointsize>23</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1회차 숫자야구</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_com1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>320</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>?</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Hidden">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>470</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(86, 235, 148);</string>
    </property>
    <property name="text">
     <string>         COMPUTER BALL</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
